Modules/TRD/src/TrackingTask.cxx

```diff
@@ -199,7 +204,7 @@ void TrackingTask::buildHistograms()
   axisConfig(mTrackPhi, "#phi", "Counts", "", 1, 1.0, 1.1);
   publishObject(mTrackPhi);
 
-  mTrackPt = new TH1D("TrackPt", "p_{#it{T}} Distribution", 100, 0.0, 10.0);
+  mTrackPt = new TH1D("TrackPt", "p_{T} Distribution", 100, 0.0, 10.0);
```

Comment:
can we have a x-axis with logarithmic binning?

Reply:
Thanks Ole. added and also tested it on test qcg